packages/salesforcedx-vscode-core/src/commands/baseDeployCommand.ts

```diff
@@ -57,7 +61,17 @@ export abstract class BaseDeployExecutor extends SfdxCommandletExecutor<
     });
 
     execution.processExitSubject.subscribe(async exitCode => {
-      this.logMetric(execution.command.logName, startTime);
+      let properties;
+      const registryAccess = new RegistryAccess();
+      try {
+        const components = registryAccess.getComponentsFromPath(
+          execFilePathOrPaths
+        );
+        const metadataCount = JSON.stringify(createComponentCount(components));
+        properties = { metadataCount };
+      } catch (e) {}
```

Comment:
Since we're silently swallowing this error (generally we shouldn't but I think it's alright here) let's put a comment saying something like "registry does not handle multiple paths. only log component count for single paths".

Reply:
We should still log this error too. Outside of not supporting multiple paths this could help us find other errors with the registry being used on metadata types with different shapes that might cause problems.

Reply:
What do you think would be the best thing to log for this?

Reply:
I think you could log something similar to what Bryan suggested above, "Registry does not handle multiple paths yet"

Reply:
You should log it as an exception, similar to how we do it a few lines below in this same file (https://github.com/forcedotcom/salesforcedx-vscode/blob/dc1e637a517d70b6b199cc5a0ae08d96e4580bb1/packages/salesforcedx-vscode-core/src/commands/baseDeployCommand.ts#L83)